dbt_copilot_helper/commands/conduit.py

```diff
@@ -277,19 +313,32 @@ def start_conduit(app: str, env: str, addon_type: str, addon_name: str = None):
 
 
 @click.command(cls=ClickDocOptCommand)
-@click.argument("addon_type", type=click.Choice(CONDUIT_ADDON_TYPES))
+@click.argument("addon_name", type=str, required=True)
 @click.option("--app", help="AWS application name", required=True)
 @click.option("--env", help="AWS environment name", required=True)
-@click.option("--addon-name", help="Name of custom addon", required=True)
-def conduit(addon_type: str, app: str, env: str, addon_name: str):
+def conduit(addon_name: str, app: str, env: str):
     """Create a conduit connection to an addon."""
     check_copilot_helper_version_needs_update()
+    application = load_application(app)
 
     try:
-        start_conduit(app, env, addon_type, addon_name)
-    except InvalidAddonTypeConduitError:
+        addon_type = get_addon_type(application, env, addon_name)
+        start_conduit(application, env, addon_type, addon_name)
+    except ParameterNotFoundConduitError:
+        click.secho(
+            f"""No parameter called "/copilot/applications/{app}/environments/{env}/addons". Try deploying the "{app}" "{env}" environment.""",
+            fg="red",
+        )
+        exit(1)
+    except AddonNotFoundConduitError:
+        click.secho(
+            f"""Addon "{addon_name}" does not exist.""",
+            fg="red",
+        )
+        exit(1)
+    except InvalidAddonTypeConduitError as err:
         click.secho(
-            f"""Addon type "{addon_type}" does not exist, try one of {", ".join(CONDUIT_ADDON_TYPES)}.""",
+            f"""Addon type "{err.addon_type}" does not exist, try one of {", ".join(CONDUIT_ADDON_TYPES)}.""",
```

Comment:
Maybe something more along the lines of `Addon type "{addon_type}" is not supported, we support: {", ".join(CONDUIT_ADDON_TYPES)}.`